### PR TITLE
Added: Home Page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Features added:

1. Add a new note
2. Pin and Unpin notes
3. Edit notes
4. Delete notes
5. Local Storage for all notes

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Everything is good by my side.

File: src/Components/UI/Button/ButtonSimple.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const ButtonSimple = (props) => {
```
Comment (shruthipatil11) [inline src/Components/UI/Button/ButtonSimple.jsx:3]: Good reusability  by creating buttons as components.

File: src/Components/UI/Modal/EditNoteModal.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+              />
+            </div>
+          </div>
+          <div>
+            <div className="input-icon">
+              <textarea
+                onChange={onChangeHandler}
+                type="text"
+                name="text"
+                autoComplete="off"
```
Comment (shruthipatil11) [inline src/Components/UI/Modal/EditNoteModal.jsx:85]: autoComplete="off" is used for security reasons during productions. Good that it is been used.

File: src/Components/NewNote/NewNote.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
+.new-note-input {
+  width: 65%;
+  border-radius: 4px;
+  box-shadow: 0 3px 5px rgba(43, 43, 43, 0.214);
```
Comment (shruthipatil11) [inline src/Components/NewNote/NewNote.css:4]: try to reuse the color variables.

File: src/Components/UI/Modal/EditNoteModal.jsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    ) {
+      dispatch({ type: "hideInputField" });
+      setEditModal(false);
+    } else {
+      dispatch({ type: "hideInputWithData" });
+    }
+  };
+
+  const onChangeHandler = (e) => {
+    const value = e.target.value;
```
Comment (shruthipatil11) [inline src/Components/UI/Modal/EditNoteModal.jsx:24]: I guess "e.target" can be directly destructured

**Review by shruthipatil11 — APPROVED** [review]
UI is great and looks clean.
Code looks good.

File: src/Components/NewNote/NewNote.css (hunk 3 above)
Comment (saurabhchirde) [reply src/Components/NewNote/NewNote.css:4]: Will move all these CSS to the component library once they get reviewed.

File: src/Components/UI/Modal/EditNoteModal.jsx (hunk 4 above)
Comment (saurabhchirde) [reply src/Components/UI/Modal/EditNoteModal.jsx:24]: It will be confusing to read, if I destructure it.

**Pull request merged**